projects/hslayers/src/components/layermanager/layermanager-wmst.service.ts

```diff
@@ -56,8 +57,6 @@ export class HsLayerManagerWmstService {
         dateComponent.search('Y') > -1 ? dateComponent.search('Y') : undefined;
       const indexOfM =
         dateComponent.search('M') > -1 ? dateComponent.search('M') : undefined;
-      const indexOfW =
```

Comment:
What was wrong with week?

Reply:
@raitisbe Nothing, it was just unused.

Reply:
I see. I was just vaguely following the ISO8601 standard: P is the duration designator (for period) placed at the start of the duration representation.
Y is the year designator that follows the value for the number of years.
M is the month designator that follows the value for the number of months.
W is the week designator that follows the value for the number of weeks.
D is the day designator that follows the value for the number of days.

Reply:
No objections here. I just deleted dead code, that's it. Do you request any changes to this PR before merging?

Reply:
Ok, lets merge